io_ogre/ui/export.py

```diff
@@ -250,6 +250,33 @@ def execute(self, context):
         description='convert all textures to format',
         default=config.get('FORCE_IMAGE_FORMAT') )
 
+    EX_EXPORT_ENABLE_LOGGING = BoolProperty(
```

Comment:
please check the currently selected tool and hide the non applicable options

Reply:
Yep, makes sense. I'll dig into the Blender APIs to see what I can do

Reply:
88fb532 now shows the detected converter (if any) and enables the related settings based on how they're named.
When prefixed by `EX_V1_` the setting is shown only for `OgreXMLConverter`, when `EX_V2_` the setting is visible only for `OgreMeshTool`, otherwise the setting is available for both converters.